### PR TITLE
Maintenance: Improve content of "Connect » Python" section

### PR DESCRIPTION
**Pull request by amotl**
## About

A few content edits where flaws have been spotted while proof-reading GH-9.

## Preview

https://crate-cloud--10.org.readthedocs.build/en/10/howtos/connect-to-cluster/python.html

### CODE REVIEW

File: docs/howtos/connect-to-cluster/python.rst [hunk 1]
```diff
@@ -26,16 +28,13 @@ Example implementation will look like this:
 	    result = cursor.fetchone()
 	    print(result)
 
-See full documentation :ref:`here <crate-python:index>`.
-
 psycopg2
 --------
```
Comment (amotl) [inline docs/howtos/connect-to-cluster/python.rst:32]: The new `psycopg3` aka. `psycopg` is here, and should be added everywhere where relevant.

## References
- https://www.psycopg.org/psycopg3/docs/
- https://github.com/crate/crate-qa/tree/master/tests/client_tests/python/psycopg3

Comment (hlcianfagna) [reply docs/howtos/connect-to-cluster/python.rst:32]: Should we go ahead and change this to just `psycopg` maybe?

Comment (amotl) [reply docs/howtos/connect-to-cluster/python.rst:32]: Converged it to https://github.com/crate/crate-clients-tools/issues/34, in order to improve the section about `psycopg3` later.

**Comment by amotl** [comment]
This patch has been superseded by GH-11.

**Pull request closed without merging**